### DISCUSSION TITLE
Choose Not Reset Filter

### DISCUSSION THREAD

**Discussion by HaoboZ** [Andrewthe13th/Inventory_Kamera#532] [Ideas or Feature requests]
Is it possible to add an option to when scanning artifacts to not reset the filter? Mostly just a speed thing, for example I'm only strongboxing 1 artifact type but it needs to scan through the whole list to get an updated list which is pretty slow if you have a lot of artifacts. Instead if it doesn't reset the filter, I can have it just scan 1 artifact set and update that to the site I use. Just an option so the default behavior doesn't change.

**Comment by mokahless** [comment]
Did something get done to this? I'm having the exact opposite issue. Default settings and it didn't reset my artifact filters before scanning.
